monitoring/post_to_slack/post_to_slack.py

```diff
@@ -84,47 +90,114 @@ def human_reason(self):
             else:
                 return f'The ALB spotted multiple 500 errors ({value}) in {service} {display_time}.'
 
-    def cloudwatch_url(self):
+    # Sometimes there's enough data in the alarm to make an educated guess
+    # about useful CloudWatch logs to check, so we include that in the alarm.
+    # The methods and properties below pull out the relevant info.
+
+    @property
+    def cloudwatch_search_terms(self):
+        """
+        Returns a list of potentially useful search terms in CloudWatch.
+        """
+        if self.name == 'loris-alb-target-500-errors':
+            return ['"HTTP/1.0 500"']
+        elif self.name.startswith('lambda'):
+            return ['Traceback', 'Task timed out after']
+        elif self.name.startswith('api_'):
+            return ['"HTTP 500"']
+        else:
+            return []
+
+    @property
+    def cloudwatch_log_group(self):
         """
-        Sometimes there's enough data in the alarm to make an educated guess
-        about useful CloudWatch logs to check.  This method tries to do that.
+        Returns the CloudWatch log group most likely to contain the error.
         """
+        if self.name == 'loris-alb-target-500-errors':
```

Comment:
Can we generate a map of name to log group using the info in the terraform state somehow?